### PR TITLE
three column layout - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-06-30 at 11 35 55 PM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.header {
+  justify-content: center;
+  padding-top: 40px;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+  margin: 30px;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:27]: Don't pollute your layout classes. The only styling on `column` classes should be `flex-basis`. If you need this, you should apply it to another class you can add in addition to the column class.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.container {
+  /* max-width: 800px; */
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:2]: You should have a max width on your `container` class

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  margin: auto;
+
+  /* justify-content: center; */
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:5]: Please remove commented code prior to submitting for review.

File: css-three-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.image {
+  width: 80px;
+  height: 80px;
+  justify-content: center;
+  margin: auto;
+
+  /* position: absolute;
+  margin-top: 24px; */
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:56]: Please remove commented code prior to submitting for review.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-02 at 4 50 00 PM

File: css-three-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.header {
+  justify-content: center;
+  padding-top: 40px;
+  letter-spacing: 1px;
+}
+
+.flex {
+  flex-basis: calc(100% / 3);
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:25]: Why isn't this on a rule for `column-third`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
My prior feedback was not addressed.

**Comment by yuliia-p** [comment]
Fingers crossed
Screen Shot 2022-07-06 at 10 03 09 PM
d

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Yuliia! Your headline is misaligned with the content below it, please revise and submit a new screenshot with your revision

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-07 at 2 22 46 PM

**Pull request merged**